### DISCUSSION TITLE
MAIL SENT BUT NOT DELIVERED, FEEDBACK SAYS SENDER REJECTED.

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2308] [Help with using Postal]
Kindly help please, I have two servers from the same host , installed postal on the two servers separately, sent the same message to same email , and I discovered one delivered without a glitch but the second one did not deliver. It shows SENT but the feedback shows SENDER REJECTED.
      I checked the postal logs for the two and I didn’t see any difference in the two logs. Does anyone know what could be wrong please? Or as anyone encountered this before. [MAIL SENT BUT NOT DELIVERED, FEEDBACK SAYS SENDER REJECTED. 
PS: if you can please provide me with the command to check the smtp_server logs separately and also MySQL logs too, perhaps I can find something helpful there. Below are screenshots for the two mails. THANKS in advance.
![mail accepted](https://user-images.githubusercontent.com/96216385/210549077-a474aca4-a149-4f7a-98f9-679378867f2f.png)
![sender rejected](https://user-images.githubusercontent.com/96216385/210549095-2e4b4107-43aa-4273-a1d4-8c66b61e5140.png)

**Comment by willpower232** [comment]
As far as Postal can tell, you have sent the messages and everything is fine. The status code 250 means "everything is okay".

I presume you will have to ask them to have a word with their spam filter.

**Comment by willpower232** [comment]
Although it seems to be having a bad time verifying your return path so maybe you should also double check that.

**Reply by Itseric100** [reply]
Thanks for the help, i have tried manipulating the return path and i got a soft fail with sender rejected AUP#DNS, so i dont think the return path is the problem anymore here, can you kindly help with the command to check the mysql logs as well as the smtp_server. Thanks
